src/main/java/bio/terra/cli/command/spend/Disable.java

```diff
@@ -12,14 +14,20 @@
 public class Disable extends BaseCommand {
   @CommandLine.Mixin bio.terra.cli.command.shared.options.SpendProfileUser spendProfileUserOption;
 
-  /** Remove access to the WSM default spend profile for the given email. */
+  /** Remove access to a spend profile for the given email. */
   @Override
```

Comment:
i assume tests exists for these modified commands.  we should add tests to cover the value being provided.

Reply:
Yeah true, wasn't sure how to make the test generic for both broad and verily servers though...

we could add a value to the test configs for an alt spend profile in the env?

Reply:
Can this not be tested without the profile existing on a real server?

Reply:
my understanding is that all the tests make calls to real servers, so I'm unsure how to meaningfully test this without the profile existing

Reply:
Having a test run potentially leak unique spend profiles isn't ideal, nor is deleting a single 'test' spend profile that would never allow tests to run in parallel.

Seems like using a static 'alt' profile in all envs is the least-of-all-evils?

Reply:
How is leaking spend profiles different from leaking workspaces, resources, SAM groups, etc?

If you want a static alt profile, we should document setting it up https://github.com/DataBiosphere/terra-cli/blob/main/ADMIN.md#setup-spend-profile . 

Note that CreateProfile, DeleteProfile should be updated as part of this ticket, even if not part of this PR, and not tested. (Right?)

Reply:
My 2 cents is that it's fine to leak things in non-prod environments. In general, I think it's fine for data in non-prod environments to be "bad". Otherwise, we wouldn't be able to test creating/deleting X.

Your proposal SG. My vote is not just hard-code "test-spend-profile", not add a server setting (line in server .json files).

Reply:
Did you mean "My vote is *to* hard-code..."?

Reply:
Yes :)